habitat_baselines/common/tensorboard_utils.py

```diff
@@ -70,3 +85,56 @@ def add_video_from_np_images(
         self.writer.add_video(
             video_name, video_tensor, fps=fps, global_step=step_idx
         )
+
+
+class WeightsAndBiasesWriter:
+    def __init__(self, config, *args: Any, **kwargs: Any):
+        r"""
+        Integrates with https://wandb.ai logging service.
+        """
+        wb_kwargs = {}
+        if config.WB.PROJECT_NAME != "":
+            wb_kwargs["project"] = config.WB.PROJECT_NAME
+        if config.WB.RUN_NAME != "":
+            wb_kwargs["name"] = config.WB.RUN_NAME
+        if config.WB.ENTITY != "":
+            wb_kwargs["entity"] = config.WB.ENTITY
+        if config.WB.GROUP != "":
+            wb_kwargs["group"] = config.WB.GROUP
+        slurm_info_dict = {
+            k[len("SLURM_") :]: v
+            for k, v in os.environ.items()
+            if k.startswith("SLURM_")
+        }
+
+        self.run = wandb.init(
```

Comment:
An error will be raised here if the user does not have wandb installed but the config has wb as writer type